frontend/packages/dev-console/src/components/import/serverless/ServerlessSection.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import * as _ from 'lodash';
+import { stateToProps } from '@console/internal/reducers/features';
+import { FLAG_KNATIVE_SERVING } from '@console/knative-plugin/src/const';
+import FormSection from '../section/FormSection';
+import { CheckboxField } from '../../formik-fields';
+import './ServerlessSection.scss';
+
+type ServerlessSectionProps = {
+  flags: { [key: string]: boolean };
+};
+
+const ServerlessSection: React.FC<ServerlessSectionProps> = ({ flags }) => {
+  if (flags[FLAG_KNATIVE_SERVING]) {
+    return (
+      <FormSection title="Serverless Options" divider>
+        <button
```

Comment:
Why is this a button?  I think what you want is a label. If pf has one. I know bootstrap does.

Reply:
done